src/protobufs/gg.proto

```diff
@@ -0,0 +1,21 @@
+syntax = 'proto2';
+
+package gg.protobuf;
+
+message Function {
+  optional string exe = 1;
```

Comment:
Why are these optional? It seems like we wouldn't want default values to be put in for them.